Describe initialized(callback) (#2010)
DevExpress/devextreme-documentation: api-reference/50 Common/utils/ui/themes/initialized(callback).md

---
---
##### shortDescription
Specifies a function to be executed after a theme is loaded.

##### param(callback): function()
The function.

---

> In some situations, UI components may apply their configuration or calculate their sizes before a theme have time to load. If you encounter issues with component appearance, use **initialized(callback)** to ensure that all styles and fonts are applied:

Reviewer: Can we remove 'a theme **have** time to load'?
```suggestion
In some situations, UI components may apply their configuration or calculate their sizes before a theme is loaded. If you encounter issues with component appearance, use **initialized(callback)** to ensure that all styles and fonts are applied:
```
